ur_moveit_config/launch/ur_moveit.launch.py

```diff
@@ -147,12 +123,15 @@ def launch_setup(context, *args, **kwargs):
     )
     robot_description_semantic = {"robot_description_semantic": robot_description_semantic_content}
 
-    kinematics_yaml = load_yaml("ur_moveit_config", "config/kinematics.yaml")
+    # kinematics_yaml = load_yaml("ur_moveit_config", "config/kinematics.yaml")
+    kinematics_yaml = PathJoinSubstitution(
+        [FindPackageShare(moveit_config_package), "config", "kinematics.yaml"]
+    )
     robot_description_kinematics = {"robot_description_kinematics": kinematics_yaml}
 
-    robot_description_planning = {
-        "robot_description_planning": load_yaml_abs(str(joint_limit_params.perform(context)))
-    }
+    # robot_description_planning = {
+    # "robot_description_planning": load_yaml_abs(str(joint_limit_params.perform(context)))
+    # }
```

Comment:
Why has this been commented out? That seems to remove functionality.

Reply:
This currently make issues when loaded. The parameters can not be parsed properly.
It is related to #265 and ros-planning/moveit2_tutorials#303.

Without it everything works, but to have things fully configurable we'll have to wait for upstream fixes.